words.html

```diff
@@ -403,447 +407,445 @@
-but if <span class="pavlov">Simpleton</span> makes a mistake and cheats,
-since <span class="all_c">Always Cooperate</span> never retaliates,
-<i>it'll keep cheating them</i>.
+<span class="pavlov">Prostak</span> wygrywa!
+To dlatego, że <span class="pavlov">Prostak</span> jest w stanie wykorzystać
+<span class="all_c">Ufnego Wspólnika</span>. Obydwaj zaczynają współpracując,
+ale jeśli <span class="pavlov">Prostak</span> popełni błąd i oszuka,
+to ponieważ <span class="all_c">Ufny Wspólnik</span> nigdy się nie mści,
+<span class="pavlov">Prostak</span> <i>będzie kontynuował oszukiwanie</i>.
 </p>
 <p id="noise_evo_2_2_btn">
-Now let's try...
+To teraz spróbujmy...
 </p>
 
 <p id="noise_evo_3">
-...the same thing as before, except instead of half-<span class="all_c">Always Cooperate</span>,
-it's half-<span class="all_d">Always Cheat</span>. It's a much <i>less</i> forgiving,
-<i>more</i> hostile environment.
+...to samo co wcześniej, ale teraz zamiast połowy <span class="all_c">Ufnych Wspólników</span>,
+mamy połowę <span class="all_d">Notorycznych Oszustów</span> To dużo <i>mniej</i> wybaczające i
+<i>bardziej</i> nieprzyjazne środowisko.
 <br><br>
-Who do you think will win now? <b>Think, then PLACE YOUR BETS:</b>
+Kto według Ciebie teraz wygra? <b>Pomyśl i POSTAW SWÓJ ZAKŁAD:</b>
 </p>
 
 <p id="noise_evo_4">
-You bet on [CHAR]. Again, go through the simulation...
+Postawiłeś na [CHAR]. Ponownie uruchom symulację...
 </p>
 <p id="noise_evo_4_2_correct">
-You were right on the money -- 
+Miałeś całkowitą rację —
 </p>
 <p id="noise_evo_4_2_incorrect">
-Good guess, but someone else took the prize -- 
+Dobry strzał, ale nie trafiłeś —
 </p>
 <p id="noise_evo_4_2">
-<span class="tf2t">Copykitten</span> wins this time!
-That's surprising that with an even <i>meaner</i> starting population,
-<span class="tf2t">Copykitten</span>,
-a <i>more forgiving</i> version of <span class="tft">Copycat</span>,
-was the most successful! (note: <span class="tf2t">Copykitten</span> is so forgiving
-it doesn't even <i>entirely</i> wipe out <span class="tft">Copycat</span>. it shares room)
+Tym razem wygrywa <span class="tf2t">Papużka</span>!
+To zaskakujące, że w jeszcze bardziej <i>nieprzyjaznej</i> początkowej populacji,
+<span class="tf2t">Papużka</span> - <i>wybaczająca</i> wersja <span class="tft">Papugi</span>,
+była najbardziej skuteczna! (uwaga: <span class="tf2t">Papużka</span> jest tak pobłażliwa,
+że nawet <i>nie pozbyła się</i> wszystkich <span class="tft">Papug</span>)
 <br><br>
-In this case, a bit of "miscommunication" (5% chance of mistake each round) could lead
-to more <i>forgiveness</i>. But is this true for <i>all</i> levels...
+W tym przypadku mała ilość "nieporozumień" (5% szansa na błąd podczas każdej rundy), mogła spowodować
+więcej <i>wyrozumiałości</i>. Ale czy tak samo jest dla <i>każdej</i>...
 </p>
 <p id="noise_evo_4_2_btn">
-...of miscommunication?
+...ilości nieporozumień?
```

Comment:
Zapytałam koleżanki, która żyje z korektur od wielu lat i mówi tak:
<img width="630" alt="zrzut ekranu 2017-08-20 o 10 10 22" src="https://user-images.githubusercontent.com/31127204/29493097-dd09c4a6-858f-11e7-91b4-f3d7963728e7.png">


 Moim zdaniem, jeśli zmieniamy ich zawartość to znaczy, że są policzalne :D